ssh-allow-friend

```diff
@@ -160,6 +160,7 @@ setup () {
 
         mkdir -p $HOME/.ssh/
         echo $USERKEY >> $HOME/.ssh/authorized_keys
+        sed -i 's/ ssh/\nssh/g' $HOME/.ssh/authorized_keys
```

Comment:
1) I am unsure which service is posting keys in the same line, as I tested [here](https://github.com/chadkouse.keys) it seems to be multiline, can you show us a example case? 
2) Can we add support to other keys type? Accordingly to http://man.he.net/man5/authorized_keys.

I'm glad for your help.
Thanks.